src/API/Middlewares/HttpExceptionsMiddleware.ts

```diff
@@ -1,23 +1,33 @@
-import * as Koa from 'koa'
-
 import { IllegalArgumentException, NotFoundException } from '@po.et/poet-js'
+import { cond, T } from 'ramda'
 
-export const HttpExceptionsMiddleware: Koa.Middleware = async (context: any, next: () => Promise<any>) => {
```

Comment:
Instead of removing the type signature, try declaring the mocks of the context as `any` in the unit tests.

```ts
const ctx: any = {}
await HttpExceptionsMiddleware(ctx, next)
```

That way we'll keep the type-safety in the code that reaches production while avoiding the need to mock the entire context object for the integration tests.

Reply:
I will try it :)

Reply:
done

Reply:
I think there's an accidental extra white space there

Reply:
yes, fixed ;)